File: Notepad.vb
Imports System.IO

Public Class Notepad
    Public fullName As String = ""
    Public dir As String = "System"
    Dim amountPressed = 0
    Private Sub Button3_Click(sender As Object, e As EventArgs) Handles Button3.Click
        amountPressed += 1
        If amountPressed = 5 Then
            MDIParent1.closeToDebug = 1
        End If
        ListBox1.Show()
        ListBox1.Items.Clear()
        Dim di As New DirectoryInfo("C:\VirtualSystem\Ramenen\Groups\" & dir)
        Dim fiArr As DirectoryInfo() = di.GetDirectories()

        Dim di2 As New DirectoryInfo(di.FullName)
        Dim fiArr2 As FileInfo() = di2.GetFiles()
            Dim fri2 As FileInfo
            For Each fri2 In fiArr2
                ListBox1.Items.Add(fri2.Name)

        Next
    End Sub

    Private Sub ListBox1_SelectedIndexChanged(sender As Object, e As EventArgs) Handles ListBox1.SelectedIndexChanged
        If Not ListBox1.SelectedIndex = -1 Then
            Dim streader As New StreamReader("C:\VirtualSystem\Ramenen\Groups\" & dir & "\" & ListBox1.SelectedItem.ToString)
            RichTextBox1.Text = streader.ReadToEnd()
            streader.Close()
            ListBox1.Hide()
        End If
    End Sub

    Public Sub open(ByVal fullpath As String)
        fullName = fullpath
        Dim streader As New StreamReader(fullpath)
        RichTextBox1.Text = streader.ReadToEnd()
        streader.Close()
        ListBox1.Hide()
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        MDIParent1.TopMost = False
        Dim newFileName = InputBox("Enter the group and file name. For example: groupname\foo.bar")
        If Not newFileName.Contains("\") Then
            MsgBox("Invalid opcode.")
        Else
            If IO.Directory.Exists("C:\VirtualSystem\Ramenen\Groups\" & newFileName.Split("\")(0)) Then
                IO.File.WriteAllText("C:\VirtualSystem\Ramenen\Groups\" & newFileName.Split("\")(0) & "\" & newFileName.Split("\")(1), RichTextBox1.Text)
            Else
                MsgBox("Invalid dicode.")
            End If
        End If
        MDIParent1.TopMost = True
    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        IO.File.WriteAllText(fullName, RichTextBox1.Text)
    End Sub

    Private Sub Notepad_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    End Sub
End Class

File: Notepad.cs
using System;
using System.IO;
using Microsoft.VisualBasic;
using Microsoft.VisualBasic.CompilerServices;

namespace Ramenen_For_Work {
    public partial class Notepad {
        public string FullName { get; set; } = "";
        public string Directory { get; set; } = "System";
        private int amountPressed = 0;

        public Notepad() {
            InitializeComponent();
        }

        private void Button3_Click(object sender, EventArgs e) {
            amountPressed++;

            if(amountPressed == 5) {
                MDIParent1.CurrentInstance.closeToDebug = 1;
            }

            ListBox1.Show();
            ListBox1.Items.Clear();

            DirectoryInfo dirInfo = new($@"C:\VirtualSystem\Ramenen\Groups\{Directory}");
            DirectoryInfo[] fiArr = dirInfo.GetDirectories();

            DirectoryInfo dirInfo2 = new DirectoryInfo(dirInfo.FullName);
            FileInfo[] fiArr2 = dirInfo2.GetFiles();

            foreach(FileInfo fileInfo in fiArr2) {
                ListBox1.Items.Add(fileInfo.Name);
            }
        }

        private void ListBox1_SelectedIndexChanged(object sender, EventArgs e) {
            if (ListBox1.SelectedIndex == -1) return;

            StreamReader streader = new($@"C:\VirtualSystem\Ramenen\Groups\{Directory}\{ListBox1.SelectedItem.ToString()}");
            RichTextBox1.Text = streader.ReadToEnd();
            streader.Close();
            ListBox1.Hide();
        }


    }
}